src/index.js

```diff
@@ -32,26 +28,16 @@ import Attendance from './components/pages/Attendance/attendance';
 import MenteeAddReview from './components/pages/AddReviews/MenteeAddReview';
 import MentorAddReview from './components/pages/AddReviews/MentorAddReview';
 import PendingApplications from './components/pages/PendingApplications/PendingApplication';
-import ScheduleMeeting from './components/common/ScheduleMeeting';
 import SupportRequests from './components/pages/SupportRequests/SupportRequests';
 import TicketDashboard from './components/pages/TicketsDashboard/TicketsDashboard';
 import Analytics from './components/pages/Analytics/Analytics';
-
-import { createStore, applyMiddleware } from 'redux';
-import { Provider } from 'react-redux';
+import AppSuccess from './components/pages/RoleApply/Applications/AppSuccess';
+import ScheduleMeeting from './components/common/ScheduleMeeting';
+import Calendar from './components/common/Calendar';
+import PrivateRoute from './components/common/PrivateRoute';
+// ########################### CONFIG AND SET UP ########################### //
```

Comment:
Cleaning up all these imports by separating them out into different categories does make them easier to read and navigate.